css/index.css

```diff
@@ -136,6 +197,36 @@ p {
     margin-bottom: 20px;
     border-bottom: 1px dashed black;
 }
+.middle-content .boxes .box:nth-child(1){
```

Comment:
This is a good use of some more advanced CSS. This is what I used when I did this project a long time ago. One thing you can do is remove .middle-content from each of these. .boxes .box:nth-child() would be enough to target each individual box.